packages/blockchain-wallet-v4-frontend/src/data/analytics/sagas.ts

```diff
@@ -226,6 +228,7 @@ export default () => {
     initUserSession,
     postMessage,
     startSession,
-    stopSession
+    stopSession,
+    waitForUserId
```

Comment:
@milan-bc probably don't need to export an imported function

Reply:
yes, we do not need it